src/main/java/ch/njol/skript/expressions/ExprFilter.java

```diff
@@ -142,20 +142,6 @@ public String toString(Event event, boolean debug) {
 		return String.format("%s where [%s]", objects.toString(event, debug), rawCond);
 	}
 
-	@Override
-	public boolean isLoopOf(String s) {
-		for (ExprInput<?> child : children) { // if they used player input, let's assume loop-player is valid
-			if (child.getClassInfo() == null || child.getClassInfo().getUserInputPatterns() == null)
-				continue;
-
-			for (Pattern pattern : child.getClassInfo().getUserInputPatterns()) {
-				if (pattern.matcher(s).matches())
-					return true;
-			}
-		}
-		return objects.isLoopOf(s); // nothing matched, so we'll rely on the object expression's logic
-	}
-
```

Comment:
Did you reimplement this somewhere? I'm not seeing it being reimplemented anywhere with the new API?

Reply:
This does not need to be reimplemented as it results in really odd behavior (and was only added as a "just in case" with no real reason)

See this convo: https://github.com/SkriptLang/Skript/pull/4963#discussion_r956754210